base/setup/welcome/welcome.c

```diff
@@ -210,6 +210,106 @@ VOID TranslateEscapes(IN OUT LPTSTR lpString)
     }
 }
 
+/*
+ * Expands the path for the ReactOS Installer "reactos.exe".
+ * See also base/system/userinit/userinit.c!StartInstaller()
+ */
+VOID ExpandInstallerPath(IN OUT LPTSTR lpInstallerPath, IN SIZE_T PathSize)
+{
+    HANDLE hFind;
+    WIN32_FIND_DATA FindData;
+    SYSTEM_INFO SystemInfo;
+    PTSTR ptr;
+
+#if 0
```

Comment:
Debug leftovers? otherwise there could be a comment specifying why this is here, and why it's disabled...

Reply:
Actually it's because I was hesitating between putting that check here, or keeping the similar check where that function is called in the code.